README.md

```diff
@@ -2507,6 +2507,21 @@ Get a general event from XSC matching the provided msi.
 event, err := xscManager.GetAnalyticsGeneralEvent(msi)
 ```
 
+#### Send error log
```

Comment:
Send error log --> Send Error Log

This section should also be added the table of contents.